metricbeat/docs/modules/openai.asciidoc

```diff
@@ -0,0 +1,74 @@
+////
+This file is generated! See scripts/mage/docs_collector.go
+////
+
+:modulename: openai
+:edit_url: https://github.com/elastic/beats/edit/main/x-pack/metricbeat/module/openai/_meta/docs.asciidoc
+
+
+[[metricbeat-module-openai]]
+[role="xpack"]
+== openai module
+
+beta[]
+
+This is the openai module.
+
+
+
+:edit_url:
+
+[float]
+=== Example configuration
+
+The openai module supports the standard configuration options that are described
+in <<configuration-metricbeat>>. Here is an example configuration:
+
+[source,yaml]
+----
+metricbeat.modules:
+- module: openai
+  metricsets: ["usage"]
+  enabled: false
+  period: 1h
+
+  # # Project API Keys - Multiple API keys can be specified for different projects
+  # api_keys:
+  # - key: "api_key1"
+  # - key: "api_key2"
+
+  # # API Configuration
+  # ## Base URL for the OpenAI usage API endpoint
+  # api_url: "https://api.openai.com/v1/usage"
+  # ## Custom headers to be included in API requests
+  # headers:
+  # - "k1: v1"
+  # - "k2: v2"
+  ## Rate Limiting Configuration
+  # rate_limit:
+  #   limit: 60 # requests per second
```

Comment:
Is this to be changes to 12 as well ?
Why have we changed the limit from 60 to 12 ?
I thought 60 was the agreed upon limit ?

Reply:
I was testing everything from scratch today and that too thoroughly. Noticed a slower rate of firing of requests. So, understanding of `limit` and `burst` was confusing which I have corrected now.

This part of the doc needs to be updated with `make update`; I will run that. Rest all doc files are updated.

```
The rate limiter works as follows:

limit: 12 means one request every 12 seconds (60 seconds / 5 requests = 12 seconds per request)
burst: 1 means only 1 request can be made in burst
This ensures you never exceed 5 requests per minute
```

So nothing changed. It's just that it wasn't configured properly by default. Rate limit is still 5 req/ min as per OpenAI.